src/server/pfs/server/driver.go

```diff
@@ -924,7 +923,7 @@ func (d *driver) inspectCommit(ctx context.Context, commit *pfs.Commit, wait pfs
 		if err != nil {
 			return nil, err
 		}
-		commitInfo.SizeBytes = uint64(size)
+		commitInfo.Details = &pfs.CommitInfo_Details{SizeBytes: uint64(size)}
```

Comment:
We should probably check if details is not nil here before setting it. If it is not nil, we should just set the size. I don't think this is a problem as is, but could be later if we fill out more details elsewhere.